components/dashboards-web-component/src/designer/components/WidgetOptionsConfiguration.jsx

```diff
@@ -17,28 +17,143 @@
  */
 
 import React, { Component } from 'react';
+
+import { FormattedMessage } from 'react-intl';
+
 import PropTypes from 'prop-types';
 import _ from 'lodash';
+import { Card, CardHeader, CardText, Checkbox, MenuItem, SelectField, TextField } from 'material-ui';
+
+const labelStyle = {
+    fontSize: 12,
+};
 
 export default class WidgetOptionsConfiguration extends Component {
     constructor(props) {
         super(props);
-
         this.doesSelectedWidgetHasOptions = this.doesSelectedWidgetHasOptions.bind(this);
+        this.getCurrentWidgetOptionsInputs = this.getCurrentWidgetOptionsInputs.bind(this);
+        this.handleWidgetOutput = this.handleWidgetOutput.bind(this);
+        this.state = {
+            options: props.selectedWidget.props.configs.options,
+        };
+    }
+
+    getCurrentWidgetOptionsInputs() {
+        const CurrentWidgetOptionsInputs = [];
+        const options = this.props.selectedWidgetConfiguration.configs.options;
+        if (options) {
+            for (let i = 0; i < options.length; i++) {
+                const value = this.state.options[options[i].id];
+                switch (options[i].type.name.toUpperCase()) {
+                    case 'TEXT':
+                        CurrentWidgetOptionsInputs.push(
+                            <div className="options-list">
```

Comment:
Where is `options-list` CSS class is defined. Seems like it is not, then shall we remove it.

Reply:
It is available in the https://github.com/wso2/carbon-dashboards/blob/master/components/dashboards-web-component/src/designer/DashboardDesigner.css#L120. Since these files are not removed yet after rewriting the components, it is still there. But we do not use these files with the new components, So I have removed the class name